### PR TITLE
Import of the widgets

### PR DESCRIPTION
**Pull request by Juliana-Dimeglio**
Import of the widgets from movie_widget in movie_project

https://user-images.githubusercontent.com/85245831/128384977-f99d2d05-93f1-4c95-b5a9-659d495bed73.mp4

### CODE REVIEW

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

File: lib/src/ui/movie_detail_screen.dart [hunk 1]
```diff
@@ -25,81 +25,39 @@ class MovieDetailScreen extends StatelessWidget {
...
   Widget build(BuildContext context) {
     return Scaffold(
-      body: SingleChildScrollView(
-        child: Column(
-          children: [
-            Container(
-              padding: EdgeInsets.only(
-                top: NumericConstants.paddingEdit,
+      body: Container(
+        decoration: BoxDecoration(gradient: backgroundColor),
```
Comment (MarcosCGlobant) [inline lib/src/ui/movie_detail_screen.dart]: missed a trailing comma here

**Review by MarcosCGlobant — APPROVED** [review]
Just one trailing comma, but LGTM

File: lib/src/ui/movie_detail_screen.dart (hunk 1 above)
Comment (Juliana-Dimeglio) [reply lib/src/ui/movie_detail_screen.dart]: Done

**Pull request merged**